### PR TITLE
Added online quiz portal #122

### PR DESCRIPTION
**Pull request by jyothi-k-g**

### CODE REVIEW

**Review by Ayushparikh-code — APPROVED** [review]

File: online quiz portal/style.css [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+
+.buttons button.quit{
+    color: #990073;
+    background: #fff;
+}
+
+.buttons button.quit:hover{
+    color: #fff;
+    background: #990073;
+}
```
Comment (chicken-biryani) [inline online quiz portal/style.css:382]: Add a new line at the end of file.

Use a css beautifier.

File: online quiz portal/script.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            clearInterval(counterLine); //clear counterLine
+        }
+    }
+}
+
+function queCounter(index){
+    //creating a new span tag and passing the question number and total question
+    let totalQueCounTag = '<span><p>'+ index +'</p> of <p>'+ questions.length +'</p> Questions</span>';
+    bottom_ques_counter.innerHTML = totalQueCounTag;  //adding new span tag inside bottom_ques_counter
+}
```
Comment (chicken-biryani) [inline online quiz portal/script.js:212]: Add a new line at the end of file.

Use a js beautifier.

File: online quiz portal/question.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    answer: "Empty set",
+    options: [
+      "0 < x < π",
+      "Empty set",
+      "0 < x < π",
+      "2nπ < x < (2n + 1) π , for n in N "
+    ]
+  },
+    
+];
```
Comment (chicken-biryani) [inline online quiz portal/question.js:47]: Add a new line at the end of file.

Use a js beautifier.

**Review by chicken-biryani — COMMENTED** [review]
Add project name in form of link and description in index.md
Please use the following tools to increase the readability of your README / Scripts before creating a pull request.
It is a good practice.

Markdown - https://codebeautify.org/markdown-formatter
HTML - https://codebeautify.org/htmlviewer
CSS - https://codebeautify.org/css-beautify-minify
JS - https://codebeautify.org/jsviewer
Use this PR as a reference :
#93

**Comment by jyothi-k-g** [comment]
@chicken-biryani  I have done all the required changes which was mentioned could you please review it

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Add your details to the index.md file. @jyothi-k-g

**Comment by jyothi-k-g** [comment]
@harshita2216  What are the details that should be added in index.md file?

**Comment by harshita2216** [comment]
> @harshita2216 What are the details that should be added in index.md file?

Please see the index.md file. 
Name, Link and a small description, the way others have done.

**Comment by jyothi-k-g** [comment]
> > @harshita2216 What are the details that should be added in index.md file?
> 
> Please see the index.md file.
> Name, Link and a small description, the way others have done.

I have included the details but as for the link should I include the link of my repository?

**Comment by harshita2216** [comment]
> > > @harshita2216 What are the details that should be added in index.md file?
> > 
> > 
> > Please see the index.md file.
> > Name, Link and a small description, the way others have done.
> 
> I have included the details but as for the link should I include the link of my repository?

Refer to the index.md file and the details that others have put.

**Comment by jyothi-k-g** [comment]
> > > > @harshita2216 What are the details that should be added in index.md file?
> > > 
> > > 
> > > Please see the index.md file.
> > > Name, Link and a small description, the way others have done.
> > 
> > 
> > I have included the details but as for the link should I include the link of my repository?
> 
> Refer to the index.md file and the details that others have put.

Others have included the links of main open source repository which include their project in that.

**Comment by jyothi-k-g** [comment]
I can only copy the link from my own repository which include the project and paste it in index.md file

**Pull request merged**